hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/io/compress/CodecPool.java

```diff
@@ -150,7 +150,9 @@ public static Compressor getCompressor(CompressionCodec codec, Configuration con
     Compressor compressor = borrow(compressorPool, codec.getCompressorType());
     if (compressor == null) {
       compressor = codec.createCompressor();
-      LOG.info("Got brand-new compressor ["+codec.getDefaultExtension()+"]");
+      if(LOG.isDebugEnabled()) {
```

Comment:
no need to wrap, just use {} so that the toString() call on codec.getDefaultExtension is only called once. yes, I know about the old code, but that's old code

Reply:
actually we can move to full slf4j here, so 

```
LOG.debug("Got brand-new compressor [{}]", codec.getDefaultExtension());
```

I've looked at the codecs and apart from one case, `PassthroughCodec`, this is all a low cost invocation so no need to wrap. but that does log at info.

so there is a good argument for guarding the logging.

but: can you add a space between the `if` and the `(`